DigitalLearningSolutions.Data/Services/SupervisorService.cs

```diff
@@ -320,6 +321,26 @@ public IEnumerable<SupervisorDashboardToDoItem> GetSupervisorDashboardToDoItems(
                 WHERE (sd.SupervisorAdminID = @adminId) AND (casv.Verified IS NULL)", new { adminId }
                 );
         }
+        public IEnumerable<SupervisorDashboardToDoItem> GetSupervisorDashboardToDoItemsForRequestedReviews(int adminId)
+        {
+            return connection.Query<SupervisorDashboardToDoItem>(
+                @"SELECT ca.ID, sd.ID AS SupervisorDelegateId, c.FirstName + ' ' + c.LastName AS DelegateName, sa.Name AS ProfileName, MAX(sasv.Requested) AS Requested, 0 AS SignOffRequest, 1 AS ResultsReviewRequest
```

Comment:
Further testing suggests there is a problem with this query. I requested verification for 1 result in a second self assessment. This appeared on my supervisor To Do list three times:
![image](https://user-images.githubusercontent.com/67740339/144451833-06ebcc95-882e-4db3-a9e5-2b6f9101fe86.png)

Reply:
The return from this query has the following return on my Dev DB for adminId = 1:
![image](https://user-images.githubusercontent.com/67740339/144452707-b839adbc-f4be-4ae9-944c-5f0fd5f7367e.png)
Note the different IDs (CandidateAssessment.IDs?) in the first column. Suggesting a problem with a join.